specification/apicenter/DataApi/models/exportSpecification.tsp

```diff
@@ -0,0 +1,10 @@
+namespace ApiCenter.DataApi.Models;
+
+@doc("Export specification")
+model ExportSpecification {
+  @doc("The result of the specification export.")
+  value: string;
+
+  @doc("The result format of the specification export.")
+  format: string;
```

Comment:
What are possible values? This should probably be declared as an enum.

Reply:
Fixed in latest revision.